CMakeLists.txt

```diff
@@ -64,6 +66,7 @@ install(PROGRAMS
 )
 
 set_target_properties(${PROJECT_NAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
```

Comment:
Do we still need this .? adding `SRDFDOM_EXPORT` to `SRDFWriter` should make this obsolete, right .?

Reply:
Yeah, I think that should make it obsolete. We would need to add `SRDFDOM_PUBLIC` because the export should only occur while building and it should import when another library includes it.

However, you would need to manually add `SRDFDOM_PUBLIC` to all new classes, and that might be easy to forget. I still think the global export should be used since it's much simpler than tagging all new classes and more closely matches the behavior of other compilers.